docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.ts

```diff
@@ -0,0 +1,384 @@
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (PPT)',
+                content: this.modalPpt
+            }
+        }, {
+            id: 15,
+            name: chance.name(),
+            dateString: '4 July 2017',
+            document: 'Document 15.doc',
+            extension: '.doc',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (DOC)',
+                content: this.modalDoc
+            }
+        }, {
+            id: 16,
+            name: chance.name(),
+            dateString: '6 July 2017',
+            document: 'Document 16.pdf',
+            extension: '.pdf',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (PDF)',
+                content: this.modalPdf
+            }
+        }, {
+            id: 17,
+            name: chance.name(),
+            dateString: '23 June 2017',
+            document: 'Document 17.doc',
+            extension: '.doc',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (DOC)',
+                content: this.modalDoc
+            }
+        }, {
+            id: 18,
+            name: chance.name(),
+            dateString: '15 June 2017',
+            document: 'Document 18.ppt',
+            extension: '.ppt',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (PPT)',
+                content: this.modalPpt
+            }
+        }, {
+            id: 19,
+            name: chance.name(),
+            dateString: '10 July 2017',
+            document: 'Document 19.doc',
+            extension: '.doc',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (DOC)',
+                content: this.modalDoc
+            }
+        }, {
+            id: 20,
+            name: chance.name(),
+            dateString: '3 July 2017',
+            document: 'Document 20.pdf',
+            extension: '.pdf',
+            storage: chance.d100().toString(),
+            active: false,
+            panel: {
+                title: 'Site Detail - UX Aspects (PDF)',
+                content: this.modalPdf
+            }
+        }
+    ];
+
+    noSelectedItem = `<div *ngIf="!selectedItem" class="preview-no-record">
+    <i class="hpe-icon hpe-document"></i>
+    <h2 ng-bind="previewEmptyText" class="ng-binding">Select an item from the list</h2>
+</div>`;
+
+    constructor(colorService: ColorService) {
+        super(require.context('./snippets/', false, /\.(html|css|js|ts)$/));
+
+        this.sparkTrackColor = colorService.getColor('accent').setAlpha(0.2).toRgba();
```

Comment:
Use the new `theme` property on the spark charts instead of having to use variables in the class